### PR TITLE
SOLR-14072: Deprecate Blob API and runtimeLib

### PR DESCRIPTION
**Pull request by dsmiley**
Bare minimum.

### CODE REVIEW

**Comment by noblepaul** [comment]
+1

File: solr/CHANGES.txt [hunk 1]
```diff
@@ -183,6 +183,10 @@ Upgrade Notes
 
 * SOLR-14065: VelocityResponseWriter has been deprecated and may be removed in a future version.
 
+* SOLR-14072: The "Blob Store" API and "runtimeLib" plugin mechanism that uses it is now considered deprecated.  The
+  replacement to it is the "Package Manager" system, which includes a "File Store".  These are experimental currently
```
Comment (chatman) [inline solr/CHANGES.txt]: Lets call it "package store".

File: solr/CHANGES.txt [hunk 2]
```diff
@@ -183,6 +183,10 @@ Upgrade Notes
 
 * SOLR-14065: VelocityResponseWriter has been deprecated and may be removed in a future version.
 
+* SOLR-14072: The "Blob Store" API and "runtimeLib" plugin mechanism that uses it is now considered deprecated.  The
```
Comment (chatman) [inline solr/CHANGES.txt]: Should we even deprecare the blob store API itself? I think Metrics system uses it. @sigram any thoughts/ideas?

File: solr/CHANGES.txt (hunk 1 above)
Comment (chatman) [reply solr/CHANGES.txt]: Should we call it Package Management system? Also, I think there are two spaces before "These".

File: solr/CHANGES.txt (hunk 2 above)
Comment (chatman) [reply solr/CHANGES.txt]: I'm +1 to deprecating it, though. Please go ahead with merge.

**Pull request merged**